### DISCUSSION TITLE
Software PDA

### DISCUSSION THREAD

**Discussion by petelindsey** [aqualinkd/AqualinkD#166] [Q&A]
I have a PDA only, the RF transmitter/receiver broke and instead of replacing it, got a pi off the shelf and installed aqualinkd, worked like a champ out of the box so I can control the pool (mostly).  So first of all massive shout out to sfeakes, what you have put together is fantastic!

My only outstanding issue(s), even though the config file is set to sync the time, it doesn't / can't, normally I could just use the remote, but that is no longer an option.  I see that if I set the logging high enough I can determine what would be on the PDA screen, so if the API supported the ability to send PDA keys then I would be golden.  Also seems that it would be quite easy from that point to create a PDA emulator... 

So the actual question, how can i send PDA keys so that I can program the items that aqualinkd doesn't natively have?

Thanks!

**Comment by ballle98** [comment]
See issue #93 and associated pull request

If you want to test my fix follow the install instructions here - https://github.com/ballle98/AqualinkD/wiki#install-configuration
